app/server/ruby/lib/sonicpi/synths/synthinfo.rb

```diff
@@ -7846,7 +7920,11 @@ class BaseInfo
         :stereo_player => StereoPlayer.new,
         :blade => SynthViolin.new,
         :piano => SynthPiano.new,
+<<<<<<< HEAD
         :rodeo => SynthRodeo.new,
+=======
+        :kalimba => SynthKalimba.new,
+>>>>>>> 8866bc098... Add kalimba synth (African thumb piano)
```

Comment:
Looks like you missed a conflict that needs resolving:
```suggestion
        :rodeo => SynthRodeo.new,
        :kalimba => SynthKalimba.new,
```

Reply:
Thanks! My editor cached a fixed version of the file so I didn't see this. Not sure how that happened. I've fixed,  rebased and pushed it again